mindjunkies/dashboard/views.py

```
@@ -1,62 +1,120 @@
+from django.shortcuts import get_object_or_404, redirect, render
 from django.urls import reverse_lazy
 from django.utils.timezone import now
-from django.views.decorators.http import require_http_methods
+from django.views import View
 from django.views.generic.edit import FormView
+from django.core.paginator import Paginator
+
 
 from mindjunkies.accounts.models import User
 from mindjunkies.courses.models import Course, Enrollment
+from mindjunkies.payments.models import Transaction, Balance, BalanceHistory
```

Ruff (0.8.2)

`mindjunkies.payments.models.BalanceHistory` imported but unused

Remove unused import: `mindjunkies.payments.models.BalanceHistory`

(F401)